docs/implementation-experience-report/README.md

```diff
@@ -0,0 +1,2 @@
+# Implementation experience report
```

Comment:
:+1:

Reply:
I also agree, but I think a few extra words would be good nonetheless. I've cooked something up. We can discuss what should be in there in the meeting in a few hours.